src/components/tables/tables.scss

```diff
@@ -1,152 +1,197 @@
 @import '../../assets/scss/variables';
 //// Table Elements
 
- table {
+.table-responsive-container {
   width: 100%;
-  font-size: $content-font-size;
-}
+  position: relative;
 
- table {
-  border-spacing: 0;
-  border-collapse: collapse;
-}
+  &.row-headers--true {
+    .track-heading:first-child {
+      left: 0;
+    }
 
- table td,
- table th {
-  padding: $md;
-  vertical-align: top;
-  padding: $md;
-}
+    &.left-sticky-minwidth .track-heading:first-child {
+      position: sticky;
+    }
+  }
 
- table th {
-  color: $secondary;
-}
+  &.is-striped {
+    &.isSticky-Left.left-sticky-minwidth {
+      table.is-striped tbody th {
+        filter: drop-shadow(0px 4px 2px rgba(0, 0, 0, 0.2));
+      }
 
- table th:not([align]) {
-  text-align: left;
-}
+      .track-heading:first-child {
+        filter: drop-shadow(0px 4px 2px rgba(0, 0, 0, 0.2));
+      }
+    }
+  }
 
- table thead td,
- table thead th {
-  color: $grey;
-  border-width: 0 0 2px;
-}
+  &.isSticky-Left.left-sticky-minwidth .track-heading:first-child {
+    border-right: 1px solid #ccc;
+  }
 
- table tfoot td,
- table tfoot th {
-  color: $grey;
-  border-width: 2px 0 0;
-}
+  .track-heading:first-child {
+    border-right: 1px solid #fff;
+    transition: border-right 0.3s ease-in-out;
+  }
 
- table tbody tr:last-child td,
- table tbody tr:last-child th {
-  border-bottom-width: 0;
+  &.isSticky-Left.is-striped.left-sticky-minwidth .track-heading:first-child {
+    border-right: none;
+  }
+
+  &.left-sticky-minwidth table tbody th {
+    position: sticky;
+  }
 }
 
 table {
-  color: $secondary;
-  background-color: $white;
+  width: 100%;
+  font-size: $content-font-size;
+  border-spacing: 0;
 }
 
-table td,
-table th {
-  //vertical-align: top;
-  //border: 1px solid $light;
-  //border-width: 0 0 1px;
- // @include padding($top: $xsm, $right: $sm, $bottom: $xsm, $left: $sm);
-}
+.headers-table {
+  position: sticky;
+  z-index: 1;
+  top: 0;
 
-table th {
-  color: $secondary;
-}
+  .scroller {
```

Comment:
There is nothing about this class name that really helps me to understand what its relationship is to the rest of the table. I would like to see this re-written to take advantage of a BEM style class naming. So that it is more clear at a glance what is happening.